source/drivers/swift.txt

```diff
@@ -0,0 +1,81 @@
+.. _swift-language-center:
+
+.. include:: /includes/unicode-checkmark.rst
+
+====================
+MongoDB Swift Driver
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+This is the official MongoDB Swift Driver.
+
+.. admonition:: Alpha Release
+   :class: important
+
+   The MongoDB Swift Driver is currently in alpha.
+
+- `General documentation for the MongoDB Swift Driver <https://mongodb.github.io/mongo-swift-driver>`__
```

Comment:
Nitpick: These bullet points could probably match the other driver documentation format better:

- Getting Started
- API Reference
- Changelog
- Source Code

Also, you probably do not have to restate it's for Swift since the title of the page is MongoDB Swift Driver.

Reply:
It looks like your bullet points were taken from the Go driver; I was referring to C, C++, C#, Java, Node, Perl, PHP, and Scala. While not necessary, staying consistent with how we label different types of documentation might be helpful to the documentation authors.